server/src/lib/utility.ts

```diff
@@ -1,13 +1,35 @@
 const md5 = require('js-md5');
 const DEFAULT_SALT = '8shd9fg3oi0fj';
+const SENTENCE_SALT = '8hd3e8sddFSdfj';
+import * as crypto from 'crypto';
 
 /**
- * Hash the string.
+ * Default hashing function
  */
 export function hash(str: string, salt?: string): string {
   return md5(str + (salt || DEFAULT_SALT));
 }
 
+/**
+ * Needs to behave like the client side hash() in /web/src/utility.ts
+ */
+export function hashClientId(text: string) {
+  return crypto
+    .createHash('sha256')
+    .update(text)
+    .digest('hex');
+}
```

Comment:
I'm actually going to leave this here for now, because this is only used to send a hashed client_id to Google Analytics in both the front and the back-end, and it's not the same hash as what the common voice bundler uses anyway, so it might make more sense to just stop sending any identifying information to GA instead of maintaining this delicate balancing act.